src/Vector3.php

```diff
@@ -421,6 +429,14 @@ public static function max(Vector3 ...$positions) : Vector3{
 		return new Vector3(max($xList), max($yList), max($zList));
 	}
 
+	/**
+	 * Returns a new vector with the minimum xyz coordinates based from the passed vectors.
+	 * Behavior is similar with the standard PHP min function < https://php.net/manual/en/function.min.php >
```

Comment:
I don't think it is in any sense similar to the PHP min function. Removing this line may lead to less confusion. PHP min function returns one of the inputs, while this doesn't.